indigo_app/views/places.py

```diff
@@ -743,16 +743,14 @@ def get_queryset(self):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-
         context['form'] = self.form
-        # TODO
         works = context["works"]
+        context['total_works'] = Work.objects.filter(country=self.country, locality=self.locality).count()
         context['page_count'] = DocumentMetrics.calculate_for_works(works)['n_pages'] or 0
```

Comment:
@longhotsummer I can't seem to get this to work locally, I dont have the DocumentMetrics objects